build.gradle

```diff
@@ -305,9 +305,13 @@ def parsePmdReport(File reportFile) {
 }
 
 def checkSubprojects = [
-        project(':kieker-common'), project(':kieker-monitoring'), project(':kieker-analysis'),
+        project(':kieker-common'),
+	project(':kieker-monitoring'),
```

Comment:
These two lines seem to have a different indentation than the others

Reply:
Sorry. Was a tabs and spaces error.